python/cudf/cudf/core/column/numerical_base.py

```diff
@@ -145,9 +144,11 @@ def quantile(
         else:
             no_nans = self.nans_to_nulls()
             # get sorted indices and exclude nulls
-            indices = sorting.order_by(
-                [no_nans], [True], "first", stable=True
-            ).slice(no_nans.null_count, len(no_nans))
+            indices = (
```

Comment:
Why this change?

Reply:
`ColumnBase.argsort` just calls `sorting.order_by`, so avoiding redefining the argsort definition here

Reply:
Ah cool, thanks